### PR TITLE
feat: add version info for offline usage

### PR DESCRIPTION
**Pull request by Klaus-Keller**
Adding package and version information which can be used to create a bundle for offline usage of SAP Fiori tools.

### CODE REVIEW

**Review by zdravko-georgiev — APPROVED** [review]
Simple and straightforward

File: docs/tools-offline-instructions/info/projectModules.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    "@sap/ui5-builder-webide-extension": [
+        "^1.1.8"
+    ],
+    "@sap/ux-specification": [
+        "1.120.8",
+        "1.71.106",
+        "1.84.86",
+        "1.96.60",
+        "1.102.39",
+        "1.108.27"
```
Comment (johannes-kolbe) [inline docs/tools-offline-instructions/info/projectModules.json:20]: @sap/ux-specification version `1.102` will be deprecated soon, should we remove it already?
```suggestion
        "1.108.27"
```

**Review by johannes-kolbe — APPROVED** [review]
- Provided config looks good
- No unexpected modules included
- Small comment about spec version, but can be updated once available

File: docs/tools-offline-instructions/info/projectModules.json (hunk 1 above)
Comment (Klaus-Keller) [reply docs/tools-offline-instructions/info/projectModules.json:20]: This file is automatically generated file, I'd like to avoid manual changes to this.

**Pull request merged**